packages/template-retail-react-app/app/constants.js

```diff
@@ -118,5 +118,10 @@ export const SHIPPING_COUNTRY_CODES = [
     {value: 'US', label: 'United States'}
 ]
 
+// Use this boolean to determine how a basket is created if there is none
+// true = defer it until an item is added to cart
+// false = create a basket on app's first load
+export const LAZY_BASKET_INITIALIZATION = false
```

Comment:
What do you think about naming this `LAZY_BASKET_CREATION`? I feel like initialization also covers cart fetching, which we aren't toggling. This will also allow us to have another setting called `LAZY_BASKET_LOAD` if we wanted to do that like Nate asked previously.